Imagine/Cache/Resolver/FlysystemResolver.php

```diff
@@ -132,7 +141,7 @@ public function store(BinaryInterface $binary, $path, $filter)
         $this->flysystem->put(
             $this->getFilePath($path, $filter),
             $binary->getContent(),
-            ['visibility' => $this->visibility]
```

Comment:
@lsmith77 Yup, FlySystem requires PHP > 5.4, so that's why it is excluded from 5.3 builds on travis and why the above wasn't being tested.